src/style.css

```diff
@@ -0,0 +1,36 @@
+
+.mstar {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  /* background-color: grey;
+  border: 3px solid red; */
+}
+
+.star {
+  display: flex;
+  height: 16px;
+  background-position: center;
+  /* background-color: rgb(149, 149, 97);
+  border: 1px solid rgb(20, 10, 100); */
+}
+
+.stars_star {
+  margin-right: 4px;
```

Comment:
don't forget about the last star in each row 😉

<img width="196" alt="image" src="https://user-images.githubusercontent.com/55802630/173054426-47fe4110-30b5-47b0-8f72-ff7d52f3f087.png">